recipes/xspec-models-lite/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set name = "xspec-models-lite" %}
+{% set version = "6.22" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+
+  url: https://heasarc.gsfc.nasa.gov/FTP/software/lheasoft/lheasoft6.22.1/xspec-modelsonly-v6.22.1.tar.gz
+  sha256: 98f43487043e5ce01db5b2ce6cededb1be4e62f13cfc53d99ebe2af9c1bbf81a
+  
+
+build:
+
+  skip: true  # [win]
+  number: 0
+
+
+requirements:
+
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - cfitsio
```

I think the build requirements should be:
```suggestion
  build:
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
    - {{ compiler('fortran') }}
    - make
```
The others look like `host` and `run` dependencies.